src/theme/datePicker.ts

```diff
@@ -0,0 +1,11 @@
+const datePicker = {
+  size: "xs",
```

Comment:
don't specify size here. size should be specified where it's used, because it often depends on context.